app/views/cart/show.html.erb

```diff
@@ -46,8 +46,12 @@
     <p>Total:
       <%= number_to_currency(cart.total) %></p>
     <p><%= link_to "Empty Cart", "/cart", method: :delete %></p>
-    <p><%= link_to "Checkout", "/orders/new", method: :get %></p>
+    <% if @items.keys.any? { |item| item.active? == false } %>
+      <p>Can't checkout with inactive items</p>
+    <% else %>
+      <p><%= link_to "Checkout", "/orders/new", method: :get %></p>
+    <% end %>
   <% else %>
```

Comment:
Add conditional that will only show the Checkout button when all items in the cart are active